tokio-util/src/codec/length_delimited.rs

```diff
@@ -546,12 +543,12 @@ impl Decoder for LengthDelimitedCodec {
     }
 }
 
-impl Encoder for LengthDelimitedCodec {
-    type Item = Bytes;
+impl<T> Encoder<T> for LengthDelimitedCodec where T: AsRef<[u8]> {
```

Comment:
I think it may make sense to keep this `Encoder<Bytes>`?

Reply:
Why do you think so? This is strictly more general.